### PR TITLE
One more round of touch-ups

### PR DESCRIPTION
**Pull request by kohsuke**
... as I was writing the documentation: https://github.com/launchableinc/mothership/pull/1007

### CODE REVIEW

File: launchable/commands/record/build.py [hunk 1]
```diff
@@ -13,33 +12,35 @@
...
-    'REPO_DIST like --source . '
-    'or REPO_NAME=REPO_DIST like --source main=./main --source lib=./main/lib',
+    help='path to local Git workspace, optionally prefixed by a label.  '
+    ' like --source path/to/ws or --source main=path/to/ws',
     default=["."],
     metavar="REPO_NAME",
     multiple=True
 )
-@click.option('--with-commit/--without-commit', help='', default=True)
+@click.option('--with-commits', help='We ask for your explicit consent that we do process your commits', is_flag=True)
```
Comment (ninjinkun) [inline launchable/commands/record/build.py:29]: In my idea, we should remove `--with-commit` and `--without-commit` options. I wrote the reason on the confluence https://launchableinc.atlassian.net/wiki/spaces/PRODUCT/pages/887226612/CLI+UX+improvement+proposals?focusedCommentId=908066921#comment-908066921

Comment (kohsuke) [reply launchable/commands/record/build.py:29]: Thanks. I'll see how they respond, but as I understand it, the `--with-commits` is adovacated by John as an "opt-in" mechanism to ensure people understand that we look at their commits. I don't have any strong opinion on this one way or the other, but I do want to cut v1.0 within this side of the year and write to PAs.

Is there anything I can do to get the rest of the changes in?

**Comment by ninjinkun** [comment]
Except `--with-commits` option things look good.

File: launchable/commands/record/build.py (hunk 1 above)
Comment (ninjinkun) [reply launchable/commands/record/build.py:29]: I strongly against with force `--with-commit` option. Customers don't have any choice except with `--with-commits`. That is why I insisted we should remove those options. It's to be a default. An alternative idea, I propose we keep `--with-commit` with `default=True`.

However, I don't want to block the CLI v1.0 release. Let's discuss tomorrow what the CLI will be in 1.0. We can consider the option to change it in 1.1 after the release.

Comment (kohsuke) [reply launchable/commands/record/build.py:29]: For others, when we say keep '--with-commits` with `default=True` what that means is that  the option is tolerated but meaningless. The behavior will be exactly the same regardless of whether or not the option is present.

Comment (awilkes) [reply launchable/commands/record/build.py:29]: Yeah, OK. I'm on board with removing the option altogether. @jlong indicated he was OK with it in Slack, too.

**Review by ninjinkun — APPROVED** [review]

**Pull request merged**